xwiki-platform-core/xwiki-platform-user/xwiki-platform-user-profile/xwiki-platform-user-profile-ui/src/main/resources/XWiki/XWikiUserPreferencesSheet.xml

```diff
@@ -247,10 +248,12 @@
 ###############################
 #macro (displayField $fieldName $label $defaultValue)
   #set ($hintKey = "XWiki.XWikiUsers_${fieldName}.hint")
+  #set ($escapedSelectId = $escapetool.xml("XWiki.XWikiUsers_0_${fieldName}"))
+  #set ($escapedDescriptionId = $escapedSelectId + "_description")
   &lt;dt&gt;
-    &lt;label&gt;$!escapetool.xml($services.localization.render($label))&lt;/label&gt;
+    &lt;label for="$escapedSelectId"&gt;$!escapetool.xml($services.localization.render($label))&lt;/label&gt;
     #if ($services.localization.get($hintKey))
-      &lt;span class="xHint"&gt;$!escapetool.xml($services.localization.render($hintKey))&lt;/span&gt;
+    &lt;span id="$escapedDescriptionId" class='xHint'&gt;$!escapetool.xml($services.localization.render($hintKey))&lt;/span&gt;
```

Comment:
Why did you remove the indentation here? I'm not sure we have a policy for this, but to me it seems like the rest of the code is indented when it makes sense from a Velocity point of view. Here the removed indentation makes it easy to miss the `#if`.

Reply:
Fixed in https://github.com/xwiki/xwiki-platform/pull/2217/commits/3d7d018870c801dbb1cdc741fcf5582a990f16b7